next/package.json

```diff
@@ -29,6 +29,7 @@
     "@t3-oss/env-nextjs": "^0.7.1",
     "@tanstack/react-query": "^5.8.4",
     "@tanstack/react-query-devtools": "^5.8.4",
+    "@types/react-dom": "^18.2.19",
```

Comment:
Maybe let's add `@types/react-dom` as a devDependency like the other type deps

Reply:
fixed in 7a20b97c49968db1207b08e985bf64a2aa955f24